features/step_definitions/apicast_steps.rb

```diff
@@ -13,5 +13,5 @@
 
 Given(/^the default proxy does not use apicast configuration driven$/) do
   proxy = @provider.default_service.proxy
-  proxy.update!(apicast_configuration_driven: false, sandbox_endpoint: 'https://api-2.staging.apicast.io:4443')
+  proxy.update!(apicast_configuration_driven: false, sandbox_endpoint: 'https://api-2.staging.apicast.io:4443', endpoint: 'https://api-2.staging.io:4443')
```

Comment:
would be interesting to know why this change was needed

Reply:
Indeed... the error was "Base URL can't be blank" but I wonder why the validation did not work before.

Reply:
possible that in the past that attribute was already set by previous steps?

Reply:
I don't think any step is being removed by this PR 🤔